### DISCUSSION TITLE
If I deleted my account on budibase cloud, is there any way to recover my app?

### DISCUSSION THREAD

**Discussion by zardilior** [Budibase/budibase#13131] [Help] [answered]
I had invited another admin user expected to only delete the old original user

**Comment by ConorWebb96** [comment, marked as answer]
Hi @zardilior,

If you have deleted your account there isn't anything we can do. Once an account is deleted all of the related data is also deleted.

**Reply by ConorWebb96** [reply]
In this case you'd likely need to have transferred owner ship over to the new admin user. For that you would have needed to reach out to support@budibase.com.
